pkg/backend/httpstate/backend.go

```diff
@@ -526,11 +534,11 @@ func (b *cloudBackend) parsePolicyPackReference(s string) (backend.PolicyPackRef
 	}
 
 	if orgName == "" {
-		currentUser, _, userErr := b.CurrentUser()
+		currentUser, userErr := b.currentUser(context.Background())
```

Comment:
optional nit: Any reason why this isn't just 'err' instead of 'userErr'?
(I understand that this is prior code so feel free to defer anything actionable here if you'd prefer.)

Reply:
Yeh this can just be err.